app/routes/video_routes.py

```diff
@@ -0,0 +1,151 @@
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import Blueprint, jsonify, request, make_response
+import requests
+from datetime import datetime, timedelta
+
+
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+
+@videos_bp.route("", methods=["POST"])
+def create_video():
+    request_body = request.get_json()
+
+    if "title" not in request_body:
+        return jsonify({"details": "Request body must include title."}), 400
+    if "release_date" not in request_body:
+        return jsonify({"details": "Request body must include release_date."}), 400
+    if "total_inventory" not in request_body:
+        return jsonify({"details": "Request body must include total_inventory."}), 400
+
+    new_video = Video(
+        title=request_body["title"],
+        release_date=request_body["release_date"],
+        total_inventory=request_body["total_inventory"],
+    )
+    db.session.add(new_video)
+    db.session.commit()
+
+    response_body = {
+
+
+        "id": new_video.video_id,
+        "title": new_video.title,
+        "release_date": new_video.release_date,
+        "total_inventory": new_video.total_inventory,
+
+    }
+
+    return jsonify(response_body), 201
+
+
+@videos_bp.route("", methods=["GET"])
+def get_videos():
+    videos = Video.query.all()
+    video_response = []
+
+    for video in videos:
+        video_response.append({
+            "id": video.video_id,
+            "title": video.title,
+            "release_date": video.release_date,
+            "total_inventory": video.total_inventory
+        })
+    return jsonify(video_response)
```

Comment:
If there was a helper function to return video data as a dictionary, we could use list comprehension to build the `video_response`

```suggestion
@videos_bp.route("", methods=["GET"])
def get_videos():
    videos = Video.query.all()
    video_response = [ video.to_dict() for video in videos ]
    return jsonify(video_response)
```